src/components/admin/form-components/SearchBar.jsx

```diff
@@ -97,23 +127,25 @@ export class SearchBar extends React.Component {
 
 SearchBar.propTypes = {
   handleClick: PropTypes.func.isRequired,
-  suggestions: PropTypes.arrayOf(
-    PropTypes.shape({
-      title: PropTypes.string.isRequired,
-      slug: PropTypes.string,
-      id: PropTypes.number.isRequired,
-    }),
-  ),
-  updateSuggestions: PropTypes.func.isRequired,
+  getSuggestions: PropTypes.func.isRequired,
+  mode: PropTypes.string.isRequired,
   fullWidth: PropTypes.bool,
   disabled: PropTypes.bool,
   enableAdd: PropTypes.bool,
-  category: PropTypes.string.isRequired,
+  slugify(props, propName) {
+    if (
+      props.enableAdd === true &&
+      (props[propName] === undefined || typeof props[propName] !== 'function')
```

Comment:
If the below `.isRequired` thing doesn't work then the way Facebook checks for required in the package  is `x == null` (which is equivalent to `x === undefined || x === null`), so just to stay in line with that we can change it here as well

Reply:
Not sure I completely understand you here. See my previous comment.

Reply:
What I mean is just that the way Facebook checks whether a prop is missing is `prop == null` not `prop === undefined` so I think we should stay in line with that (and you're also allowed to expand it to check both undefined and null since otherwise the linter will probably also complain)